inc/saiswitch.h

```diff
@@ -3084,6 +3084,45 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_EXTENDED_PORT_STATE_CHANGE_NOTIFY,
 
+     * @brief Tam telemetry reporting byte size of chunk
+     *
+     * Defines the maximum number of bytes in a single report.
+     * The total number of bytes in a report should be as close as possible to this value.
+     * We can increase this value to reduce the number of sys calls.
+     * If the type of message is IPFIX, this value should not be less than 65535.
+     * Because we don't expect the IPFIX record to be fragmented.
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 65535
+     */
+    SAI_SWITCH_ATTR_TAM_REPORT_CHUNK_SIZE,
```

Comment:
Please move this to the TAM_REPORT (or the TAM) object being used. It doesn't make sense as a switch attribute and moving to ta/tam_report also allows different chunk sizes per-report/per-tam if allowed by implementation.

Reply:
Actually, It was the previous design.  But we discussed with some vendors, they said they can only support global ring buffer configuration and cannot be changed at runtime.

Reply:
OK. The attribute atleast needs to be renamed to include _ST_ or equivalent since TAM is too generic and there are many other TAM features, and TAM_REPORT_CHUNK_SIZE is too generic.